File: src/main/java/org/demoth/aworlds/server2/api/messaging/fromClient/LoginMessage.java
package org.demoth.aworlds.server2.api.messaging.fromClient;

import org.demoth.aworlds.server2.api.messaging.Message;

public class LoginMessage extends Message {
    public String login;
    public String password;

    public LoginMessage() {
    }

    public LoginMessage(String login, String password) {
        this.login = login;
        this.password = password;
    }
}

File: src/main/kotlin/org/demoth/aworlds/server2/api/messaging/fromClient/LoginMessage.kt
package org.demoth.aworlds.server2.api.messaging.fromClient

import org.demoth.aworlds.server2.api.messaging.Message

class LoginMessage(var login: String, var password: String) : Message() {

    override fun toString(): String {
        return "LoginMessage{" +
                "login='" + login + '\''.toString() +
                ", password='" + password + '\''.toString() +
                '}'.toString()
    }
}
